client/src/components/Modal.js

```diff
@@ -6,6 +6,7 @@ import { useResponsive } from 'hooks/useResponsive'
 
 export const Modal = ({
   title,
+  titleNoBreakline = 'normal',
```

Comment:
I think we could remove this option at make all modal titles `no-wrap`. I don't think we would ever want these titles to wrap.

Reply:
Great point!